cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/ZookeeperLeaderFinderTest.java

```diff
@@ -63,6 +55,7 @@ public class ZookeeperLeaderFinderTest {
     private static Vertx vertx;
     private final SelfSignedCertificate zkCertificate = SelfSignedCertificate.create();
     private final SelfSignedCertificate coCertificate = SelfSignedCertificate.create();
+    private final static TlsPemIdentity DUMMY_IDENTITY = new TlsPemIdentity(null, null);
```

Comment:
Same as before. If this is needed, you should place it with the other static final fields up top.